includes/service-providers/mailchimp/class-newspack-newsletters-mailchimp.php

```diff
@@ -1426,7 +1426,7 @@ public function validate( $result, $preferred_error = null ) {
 				throw new Exception( esc_html__( 'A Mailchimp error has occurred.', 'newspack-newsletters' ) );
 			}
 		}
-		if ( ! empty( $result['status'] ) && in_array( $result['status'], [ 400, 404 ] ) ) {
+		if ( ! empty( $result['status'] ) && 400 <= (int) $result['status'] ) {
```

Comment:
Maybe it's just me, but I find this version more readable:

```suggestion
		if ( ! empty( $result['status'] ) && (int) $result['status'] >= 400  ) {
```

Reply:
Yoda conditionals you do not like, see I do! (Me too, honestly!)